app/assets/stylesheets/placement-preference.scss

```diff
@@ -0,0 +1,15 @@
+#placement-preference {
+  form {
+    label.govuk-heading-m {
+      margin-top: 30px;
```

Comment:
I'm guessing these need to use the `@include govuk-responsive-margin(??, "bottom");` then they'll adjust properly on mobile

Or do they actually need to be that set amount always? I'd rather delegate sizing to the front-end design system if we can.

Reply:
https://design-system.service.gov.uk/styles/spacing/

Reply:
The prototype had slightly more margin than what we get from the design system. I'll remove this margin and let the spacing fall through to the design system 👍

Reply:
Yeh, the prototype is currently quite 'hacked together' so don't taken any of the design as 'golden' is more for checking content, quality of question, flow. 

For the beta I'd much prefer we follow defaults unless its a conscious decision by Paul to do otherwise.

Thanks for the changes